src/main/java/org/swisspush/redisques/performance/UpperBoundParallel.java

```diff
@@ -92,10 +95,15 @@ private <Ctx> void resume(Request<Ctx> req) {
                     }
                     return;
                 }
-                if (!req.limit.tryAcquire()) {
+                if (req.numTokensAvailForOurself > 0) {
+                    // We still have a token reserved for ourself. Use those first before acquiring
+                    // new ones. Explanation see comment in 'onOneDone()'.
+                    req.numTokensAvailForOurself -= 1;
+                }else if (!req.limit.tryAcquire()) {
```

Comment:
trivial, code format

Reply:
If ok, I will do minor stuff in a separate PR.